### PR TITLE
Add Login and Logout Buttons to Home and Admin Page

### PR DESCRIPTION
**Pull request by daltonfury42**
Added user login and logout functionality.

TODO: Send JWT token on all requests
TODO: Add a loading indicator.

Note: Login won't work i preview urls, we have configured only for 
```
http://localhost:3000 
https://simplq.me 
https://www.simplq.me
```

I have tested it only on localhost, hoping that it will work for our domain.

image

image

### CODE REVIEW

File: simplq/src/styles/navbar.module.scss [hunk 1]
```diff
@@ -56,11 +56,15 @@
...
+    position: fixed;
+    // In case of smaller screens with sidebar, display at the bottom of sidebar
+    bottom: 1rem;
     @media (min-width: 787px) {
-      position: fixed;
-      top: 1.2rem;
-      right: 6rem;
+      // In case of navbar on larger screens, dispaly at top right of nav
+      top: 0.5rem;
+      right: 0rem;
```
Comment (daltonfury42) [inline simplq/src/styles/navbar.module.scss:67]: image

File: simplq/src/components/common/LoginButton.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+  const onLogoutCallback = () => {
+    dispatch(setLoggedInUser(null));
+    setLoadingIndicator(false);
+  };
+
+  if (loadingIndicator) {
+    // todo Use spinner
+    return <div>Loading...</div>;
+  }
```
Comment (maaverik) [inline simplq/src/components/common/LoginButton.jsx:33]: We should add the loading indicator component later

File: simplq/src/components/common/LoginButton.jsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      <GoogleLogout
+        render={(renderProps) => (
+          <Button color="primary" onClick={renderProps.onClick} variant="outlined">
+            <Avatar
+              id={styles.avatar}
+              alt={`${loggedInUser.givenName} ${loggedInUser.familyName}`}
+              src={loggedInUser.imageUrl}
+            />
+            &nbsp;&nbsp;Logout
+          </Button>
```
Comment (maaverik) [inline simplq/src/components/common/LoginButton.jsx:46]: This is required, right?

**Review by maaverik — APPROVED** [review]

**Comment by maaverik** [comment]
At the moment, there's only that redux change that happens on login, right?

File: simplq/src/components/common/LoginButton.jsx (hunk 3 above)
Comment (daltonfury42) [reply simplq/src/components/common/LoginButton.jsx:46]: &nbsp;&nbsp; ? Nah

**Pull request merged**